Source/FFmpegMediaSource.cpp

```diff
@@ -1094,7 +1094,7 @@ namespace winrt::FFmpegInteropX::implementation
     Windows::Media::Core::MediaStreamSource FFmpegMediaSource::GetMediaStreamSource()
     {
```

Comment:
We can't just rename a public API. I'd rather throw if this is called when mssWeak is empty.

Reply:
Actually, devs can get the MediaStreamSource from the MediaPlaybackItem as well. We could deprecate this API. It is not reliable anymore if devs use the AutoClose feature, and I don't like the idea of throwing when mssWeak is nullptr.

Reply:
Yes, I am also thinking of something along these lines. We should signal that the MediaStreamSource API is intended for transcoding work. There is confusion about that API every now and again, people trying to use the MediaStreamSource instead of MediaPlaybackItem

Reply:
It would really make sense to have a separate API for creating an MSS for transcoding (even if we use WinRT APIs). Currently, we only hand out samples for the currently selected streams. I guess if the MSS is used for transcoding and it contains multiple streams, only the default streams would be transcoded and all other streams would probably be lost.

If we'd have e.g, a separate CreateMediaStreamSourceForTranscoding, we could enable all streams for transcoding there.

But this is out of the scope of this PR. I'd say we just leave it as is here, and maybe give some more thought about how to deal best with transcoding scenarios.

Reply:
I think we would get stream switch requests with multiple streams when transcoding. But yeah, let's leave it as is for now.